reinforcement_learning/examples/basic_usage_cpp/basic_usage_cpp.h

```diff
@@ -1,3 +1,10 @@
+/**
+ * @brief Simple RL Inference API sample * 
+ * 
+ * @file basic_usage_cpp.h
+ * @author your name
```

Comment:
I guess `your name` should be `Rajan Chari`

Reply:
Done